about.html

```diff
@@ -0,0 +1,122 @@
+       <p class="paragraph1">Congratulations! We’re very excited that you will be doing Model United
+          Nations for the first time. You are about to embark on a life-changing journey that will help 
+          you develop confidence in leading others, a stronger awareness of global issues, and the chance 
+          to make new friends from around the world. Model UN is a valuable experience that can help you get
+           into college, find your first job, and become inspired to change the world.</p>
+      </div>
+      <div class="abot-div2">
+        <p>please contact us through email for further information and more about DCD FASHION .</p>
+        <p class="email">divinemaina@gmail.com</p>
+      </div>
+      <div class="abot-div3">
+        <div>
+          <h5>How do I prepare for my first participation in DCD fashion ?</h5>
+          <hr>
+          <p>Once you have signed up for a conference, you will receive your country assignment, your committee, and its topics. Most conferences provide a Background Guide or Topic Synopsis that introduces the topics
+
+            There are typically three items to prepare before you walk into your DCDfashion event: theproject you will present, your Opening Speech, and a Research Binder.</p>
+        </div>
+        <div class="about-logo">
+          <img class="about-logoimg" src="./assets/images/airbnb.png" alt="image" >
+        </div>
+        <div>
+          <h5>the past DCD fashion modeling event</h5>
+          <p>currently we have had more than seven worldwide event which were held in different countries </p>
+          <div class="events-pictures">
+            <div >
+            <div class="past-summit-overlay">
+              <h1 class="past-summit-overlay-title">2015</h1>
+              <p class="past-summit-overlay-paragraph">ladies night</p>
+            </div>
+          
+              <img  class="past-pictrs" src="./assets/images/modal3.JPG" alt="ghth" >
+            </div>
+            <div>
+              <div class="past-summit-overlay">
+                <h1 class="past-summit-overlay-title">2010
+                </h1>
+                <p class="past-summit-overlay-paragraph">gentleme's night</p>
+              </div>
+              <img  class="past-pictrs" src="./assets/images/modal5.JPG" alt="ghth" >
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="paterners">
+      <div>
+        <h5>paterners</h5>
+      </div>
+      <div class="patteners-imglogo">
+        <img  class="patterners-logo" src="./assets/images/airbnb.png" alt="" >
+        <img class="patterners-logo"  src="./assets/images/daum.png" alt="" >
+        <img class="patterners-logo"  src="./assets/images/naver.png" alt="" >
+        <img class="patterners-logo" src="./assets/images/.png" alt="" >
+        <img class="patterners-logo" src="./assets/images/.png" alt="" >
+      </div>
+     
+    </section>
```

Comment:
- [ ] This section is looking okay but as per requirements you should follow the design from [behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) which means colors, font-family, padding, margin, etc.. but you were free to personalize just contents. kindly increase your padding and margin. :innocent: 🙏🏻 
- [OPTIONAL] What if you use for the list of images a `ul` tag to group them, and each of them in a `li`
 per img;
 I know you can do it 😄 💪🏻 
- Original Design:

![part](https://user-images.githubusercontent.com/79058364/169174040-3da6b930-7e6e-4b7d-9ac7-2b33f8dc016a.png)

 - Your Page
 
![part11](https://user-images.githubusercontent.com/79058364/169174098-340d46bd-11ee-4c08-ad4c-b4b3e65c41ef.png)